draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -164,55 +164,58 @@ The YANG data model defined in this document conforms to the Network Management
 
 ## Terminology
 
-  The following terms are defined in {{!RFC7950}} and are not
-  redefined here:
+The following terms are defined in {{!RFC7950}} and are not redefined here:
+- client
+- server
+- augment
+- data model
+- data node
 
-  *  client
+The following terms are defined in {{!RFC6241}} and are not redefined here:
+- configuration data
+- state data
 
-  *  server
+The following terms are defined in {{IANA_YANG}} and are not redefined here:
```

Comment:
{{IANA_YANG}} does not have these terms!

I guess you meant [RFC 8348](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwi4w_z21aSKAxU_UaQEHQuRC98QFnoECBYQAQ&url=https%3A%2F%2Fdatatracker.ietf.org%2Fdoc%2Fhtml%2Frfc8348&usg=AOvVaw37XEzeeG-tUB3JYKmiSmSS&opi=89978449)

Reply:
My understanding is that these terms are defined in the description statements of the component classes identities within [IANA_YANG]

```suggestion
The following terms are defined in the description statements of the corresponding YANG identities, defined in {{IANA_YANG}}, and are not redefined here:
```